metastats-domain/src/main/groovy/life/qbic/metastats/filter/FilterExperimentData.groovy

```diff
@@ -5,6 +5,13 @@ import life.qbic.metastats.datamodel.MetaStatsSample
 
 interface FilterExperimentData {
```

Comment:
```suggestion
interface ExperimentDataFilter {
```

Reply:
We discussed that since it is a use case it states what is done, Filter experiment data. But if the @qbicsoftware/itss team thinks that we should not do that I can change it.

Reply:
So this is the `FilterExperimentDataInput` interface?

Reply:
Correct. I can add `Input` and change `FilterExperimentDataImpl` to `FilterExperimentData`

Reply:
I prefer to avoid `Impl` as a suffix. This is your PR and you are the author of the code so you can decide if you want to change it or not. This is mostly personal taste.